backend/dal/Helpers/Extensions/EntityExtensions.cs

```diff
@@ -136,8 +136,11 @@ public static void SetOriginalConcurrencyControlNumber<T>(this T source, DbConte
                 }
                 else
                 {
-                    context.Entry(oldItem).CurrentValues.SetValues(item);
-                    dbItemsMap.Remove(item.Id);
+                    if (updateChildValues)
```

Comment:
provide more control over the updateChild logic, to only allow updating child associations and not the contents of children.